src/schema/searchableItem.ts

```diff
@@ -36,6 +36,10 @@ export const SearchableItem = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLID),
       resolve: item => toGlobalId("SearchableItem", item._id),
     },
+    description: {
+      type: GraphQLString,
+      resolve: item => item.description,
```

Comment:
#minor this resolver is actually totally optional, since you're just destructuring a same-named key from the object.

Reply:
Ahh, good to know! Thanks!